src/store/routeAssets/getters.ts

```diff
@@ -162,6 +182,32 @@ const getters = defineGetters<RouteAssetsState>()({
       asetSymbol: maxInputAmount.assetSymbol,
     };
   },
+  outcomeAssetsAmountsList(...args): Array<OutcomeAssetsAmount> {
+    const { getters } = routeAssetsGetterContext(args);
+    const recipientsWithUsingExistingTokens = getters.recipients
+      .filter((item) => item.useTransfer)
+      .map((item) => ({ symbol: item.asset.symbol, ...item }));
+    return Object.values(groupBy(recipientsWithUsingExistingTokens, 'symbol')).map((assetArray: Array<Recipient>) => {
+      const reduceData = assetArray.reduce(
+        (acc, item) => {
+          return {
+            usd: new FPNumber(item.usd).add(acc.usd),
+            totalAmount: new FPNumber(item.amount || 0).add(acc.totalAmount),
```

Comment:
better to use predefined static value from library

`item.amount ? new FPNumber(item.amount) : FPNumber.ZERO`...